doc/INSTALL-ubuntu.md

```diff
@@ -0,0 +1,49 @@
+# Installation on Ubuntu
+
+Tested on Lubuntu Eoan (19.04) using `bash` as the shell.
+
+1. Install dependencies:
+
+        sudo apt install --install-suggests libsdl-gfx1.2-dev
+        sudo apt install qt4-default coreutils build-essential
```

Comment:
Qt4 :(

I've been meaning to nuke Qt4 support, so... uh, maybe we suggest Qt5 instead? It works great ;)